_posts/tumblr/2015-07-20-most-of-the-time-we-dont-know-what-the-hell-we're-doing-"-an-interview-with-anthony-armendariz.md

```diff
@@ -0,0 +1,67 @@
+---
+layout:    post
+title:     "'Most of the time we don't know what the hell we're doing.' -An interview with Anthony and Natalie Armendariz, partners at Funsize"
```

Comment:
Stefy, a mí me gustaría cambiarle el título, porque me da un poco de pena con FNSZ que diga "Most of the time I don't know what we're doing".

Se me ocurre algo como: 
"A conversation with Funsize: Collaboration = Fun"
"Hire your friends. A conversation with Funsize"
"Company culture at the center. A conversation with Funsize".

Pero no lo quería cambiar sin preguntarte antes qué pensabas. Qué piensas?